core/textile/secure_bucket_client.go

```diff
@@ -245,3 +268,220 @@ func (s *SecureBucketClient) decryptPathData(
 func cleanBucketPath(path string) string {
 	return strings.TrimPrefix(path, "/")
 }
+
+type pathPullingFn func(context.Context, string, string, io.Writer, ...bc.Option) (bool, error)
+
+type pullSuccessResponse struct {
+	file        *os.File
+	shouldCache bool
+}
+
+func (s *SecureBucketClient) racePullFile(ctx context.Context, key, encPath string, w io.Writer, opts ...bc.Option) error {
+	pullers := []pathPullingFn{s.pullFileFromLocal, s.pullFileFromClient, s.pullFileFromDHT}
+	// pullers := []pathPullingFn{s.pullFileFromLocal}
```

Comment:
This should be uncommented for now right?

Reply:
Fixed in a40be09